src/components/Thread/Thread.vue

```diff
@@ -674,10 +674,13 @@ export default {
         text_color (message) {
             try {
                 let colorString;
-                if (message.message_from)
-                    colorString = this.getColor(message);
-                else // Otherwise default color
+
+                if (message.type == 1) { // Message was sent by the user, use default color
```

Comment:
[nit] This should likely be an enum or const for readability

Reply:
Where is the enum defined? I can't seem to find it after a quick scan.

Reply:
Heh, sorry. It doesn't exist. I'll todo this for later.